src/biblebot/auth.py

```
@@ -0,0 +1,363 @@
+        )
+        tmp.write(data)
+        tmp.flush()
+        os.fsync(tmp.fileno())
+        tmp_name = tmp.name
+    finally:
+        if tmp:
+            tmp.close()
+    try:
+        try:
+            os.chmod(tmp_name, 0o600)
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)